frontend/src/forms/manage-restaurant-form/ManageRestaurantForm.tsx

```diff
@@ -68,6 +71,13 @@ const ManageRestaurantForm = ({ onSave, isLoading }: Props) => {
         <CuisinesSection />
         <Separator />
         <MenuSection />
+        <Separator />
+        <ImageSection />
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/31#issuecomment-2392842420>#ecff14</a><span> - 10/04/2024, 05:21 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The newly added ImageSection component is not integrated with the form's state management.
</div>
<div id="fix">
<b>Fix:</b> Integrate the ImageSection component with the form's state management using the register method from react-hook-form.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -75,1 +75,1 @@
 -        <ImageSection />
 +        <ImageSection {...form.register('image')} />
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged